src/api/app/controllers/status_messages_controller.rb

```diff
@@ -1,50 +1,38 @@
 class StatusMessagesController < ApplicationController
-  class PermissionDeniedError < APIError
-    setup 403
-  end
-
-  class CreatingMessagesError < APIError; end
+  skip_before_action :require_login, only: [:show, :index]
```

Comment:
@saraycp should we really skip this? I think we almost require to be logged in for all api endpoints (besides the /about)

Reply:
In the WebUI, non-logged-in users (nobody) can see certain kind of announcements.
AFAIR, we tried to do the same via API by adding this line.

However, now that we decided no to restrict the XML output according to communication scope, we can remove this line, otherwise non-logged-in users could see status messages published for other groups.

What do you think @vpereira and @hennevogel ?

Reply:
Ok, I just thought we limit the usage of the API as much as possible to authenticated users, I don't see a problem in making those information's available to everyone.

Reply:
Since its more a political question, its not a blocker to me :D those informations are public anyway, everyone can sign up.

Reply:
We are not trying to hide anything from anyone. We just don't want to bother people with stuff they don't need to see in the UI

In the API this is about data manipulation with some client. The API is not a UI. And @krauselukas is right, we usually only do API calls authenticated. Stick to that.

Reply:
Line dropped!